versions.tf

```diff
@@ -0,0 +1,10 @@
+terraform {
+  required_version = ">=1.0"
+
+  required_providers {
+    azurerm = {
+      source  = "hashicorp/azurerm"
+      version = "~> 2.72"
+    }
```

Comment:
main.tf uses the random_string resource.
Please exclude the provider here as required.